### PR TITLE
Preprocess progress

### PR DESCRIPTION
**Pull request by chongyi0717**
add argparse and edit source name

### CODE REVIEW

File: define.py [hunk 1]
```diff
@@ -0,0 +1,132 @@
+import json
+import supporter as sp
+from enum import auto
+from strenum import StrEnum
+
```
Comment (chongyi0717) [inline define.py:5]: source name using english name

File: preprocess_data.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import pandas as pd
+from define import *
+import os
+import argparse
+
```
Comment (chongyi0717) [inline preprocess_data.py:5]: adding argparse

File: preprocess_data.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
+import pandas as pd
+from define import *
+import os
+import argparse
+
+parser = argparse.ArgumentParser(description='parse file names')
+parser.add_argument('--input_file_ch',type=str,default='news_data/國際新聞_中文.csv')
+parser.add_argument('--input_file_en',type=str,default='news_data/國際新聞_英文.csv')
+parser.add_argument('--output_file',type=str,default='clean_news_data')
```
Comment (Nana2929) [inline preprocess_data.py:9]: It's actually `--output_dir` (directory) not a `output_file`.

**Review by Nana2929 — APPROVED** [review]
LGTM.

**Pull request merged**